### PR TITLE
fix docker events message

### PR DESCRIPTION
**Pull request by AlexeyCL**
## Description
Fixing the problem of hanging process, that caused by waiting for the error message

## Checklist
- [ ] Unit/Integration test added (if applicable)
- [ ] Documentation added/updated (wiki or md)
- [x] Code style is consistent with the rest of the project

### CODE REVIEW

**Comment by AlexeyCL** [comment]
@stsvilik I think we need to update the packages. I see an opened PR for this, due to old packages CI failed

**Review by stsvilik — COMMENTED** [review]
> @stsvilik I think we need to update the packages. I see an opened PR for this, due to old packages CI failed

Next PR will address failing tests and dependencies upgrades.

File: src/modules/dockerEvents.js [hunk 1]
```diff
@@ -38,7 +37,11 @@ const DockerEvents = {
         /**
          * Capture stderr
          */
-        const errorMessage = await readStream(ps.stderr);
+        let errorMessage = '';
+
+        ps.stderr.on('data', data => {
```
[the remark this answers is not in the document]
Comment (AlexeyCL) [reply src/modules/dockerEvents.js:42]: @stsvilik I'have checked some errors, all work good

**Comment by AlexeyCL** [comment]
> > @stsvilik I think we need to update the packages. I see an opened PR for this, due to old packages CI failed
> 
> Next PR will address failing tests and dependencies upgrades.

@stsvilik if you want I can bump all wdio dependencies. In our automation project, we'd updated to 6.10.6 and all works good

**Comment by stsvilik** [comment]
> > > @stsvilik I think we need to update the packages. I see an opened PR for this, due to old packages CI failed
> > 
> > 
> > Next PR will address failing tests and dependencies upgrades.
> 
> @stsvilik if you want I can bump all wdio dependencies. In our automation project, we'd updated to 6.10.6 and all works good

In this case I would like to keep PR's separate.

**Review by stsvilik — APPROVED** [review]

**Pull request merged**